README.md

```diff
@@ -3,61 +3,18 @@
-- [Camunda Automation Platform 7](https://github.com/camunda/camunda-bpm-platform)
-
-### Participation
-
-Anyone can participate in the issues repository to share feedback with Camunda or show support for specific features. 
-
-- *Show support for specific feature*: to show your interest, you can write a comment on an existing issue or simply give a thumb up :+1:
-- *Propose a feature*: Use the [issue template](https://github.com/camunda/issues/issues/new/choose) to propose a Feature Request. Before you do so, please check the [component repository](https://github.com/camunda/issues#public-product-components) listed above to find out if the feature already exists.
-- *Report a bug*: Use the issue template to file a Bug Report. Before you do so, make sure to check the component repository ([listed above](https://github.com/camunda/issues#public-product-components)) to find out if the bug has already been reported; and open the bug in the appropriate repository whenever possible.
-- *Change to the documentation*: Do you have suggestions to improve the documentation? Please write us in our dedicated repository, [camunda-platform-docs](https://github.com/camunda/camunda-platform-docs/), and we will make sure to follow up with you.
-
-### Disclaimer
-Any statement in this repository that is not purely historical is considered a forward-looking statement. Forward-looking statements included in this repository are based on information available to Camunda as of the date they are made, and Camunda assumes no obligation to update any forward-looking statements. The forward-looking product roadmap does not represent a commitment, guarantee, obligation or promise to deliver any product or feature, or to deliver any product and feature by any particular date, and is intended to outline the general development plans. Customers should not rely on this roadmap to make any purchasing decision.
+In this repository, you can find Camunda GitHub issues, that we selectively publish by mirroring them from internal repositories, to ensure you can follow along on updates. (Please note that most Camunda products are developed in a publish repository, so you can access issues there directly. However, some components use internal repositories, so that we use this repository to selectively publish individual issues to keep you informed. An example is Camunda Web Modeler).
+
+Please note that next to this repository, you can follow along on updates using the following links:
+
+- [Camunda Roadmap](https://roadmap.camunda.com/)
+- Public repositories. Please refer to the respective repositories and their issues to open bugs, features requests, or to learn what we are working on, on a more technical level:
+    - [Camunda 8 Orchestration cluster: Zeebe, Tasklist, Operate, Optimize, Identity, C8Run](https://github.com/camunda/camunda)
+    - [Camunda Helm](https://github.com/camunda/camunda-platform-helm)
+    - [Connectors](https://github.com/camunda/connectors)
```

Comment:
```suggestion
    - [Camunda 8 Connectors](https://github.com/camunda/connectors)
```
(for consistency with the rest of the naming)

Reply:
Valid. Corrected.